lkft-fastboot.jinja2

```diff
@@ -14,3 +14,41 @@
   series: lkft
   email-notification: {{CI_MAIL_RECIPIENTS | default('""')}}
 {% endblock metadata %}
+
+{% block deploy_target %}
+{{ super() }}
+- test:
+    namespace: tlxc
+    timeout:
+      minutes: 30
```

Comment:
please set the timeoute with variable so it's easy to calculate total job timeout